quiz/tuts/tut_p_mod1_1.php

```diff
@@ -14,15 +14,15 @@
 echo "<h2>" . sprintf(_("Moderate Proofreading Tutorial, Page %d"), 1) . "</h2>\n";
 echo "<h3>" . _("Superscripts") . "</h3>\n";
 echo "<p>" . _("Older books often abbreviated words as contractions, and printed them as superscripts. Proofread these by inserting a single caret (<kbd>^</kbd>) followed by the superscripted text. If the superscript continues for more than one character, then surround the text with curly braces <kbd>{</kbd> and <kbd>}</kbd> as well. For example:") . "</p>\n";
-echo "<table width='100%' align='center' border='1' cellpadding='4' cellspacing='0' summary='" . _("Superscripts example") . "'>\n";
-echo "  <tbody>\n";
-echo "    <tr><th align='left' bgcolor='cornsilk'>" . _("Original Image:") . "</th></tr>\n";
+echo "<table class='basic' summary='" . _("Superscripts example") . "'>\n";
+echo "  <tbody class='left-align'>\n";
+echo "    <tr><th>" . _("Original Image:") . "</th></tr>\n";
 echo "    <tr>\n";
-echo "      <td valign='top'>Gen<sup>rl</sup> Washington defeated L<sup>d</sup> Cornwall's army.</td>\n";
+echo "      <td>Gen<sup>rl</sup> Washington defeated L<sup>d</sup> Cornwallis's army.</td>\n";
```

Comment:
Was the change from `Cornwall's` to `Cornwallis's` intentional?

Reply:
Yes -- the corresponding mention in the Proofreading Guidelines was changed some time back. I verified with @lhamilton1 before I made the change here.

Reply:
Great - thanks!